contracts/TrueCurrencies/TrueUSD.sol

```diff
@@ -28,7 +28,36 @@ contract TrueUSD is TrueRewardBackedToken, DelegateERC20 {
         return "TUSD";
     }
 
-    function canBurn() internal pure returns (bytes32) {
+    function canBurn() internal override pure returns (bytes32) {
         return "canBurn";
     }
+
+    function _transferAllArgs(
```

Comment:
Is this required for the migration to 0.6 or can we avoid having to implement these functions again? If we can avoid it, add these to a new PR with the other changes to TrueUSD / TrueRewardBackedToken

Reply:
Sadly yes, it's required. TUSD extends `TrueRewardBackedToken` and `DelegateERC20` both of which implement have  `_transferAllArgs`. Solidity 0.6 wouldn't compile without this change